SOBridge_BeowulfRiflePack/Config/XComTemplateEditor.ini

```diff
@@ -0,0 +1,6 @@
+[WOTCIridarTemplateMaster.X2DLCInfo_Last]
+
+; Sets up RequiredTechs, RequiredEngineeringScore, and bVisibleIfPersonnelGatesNotMet. Resets SpecialRequirementsFn
++Edit_X2WeaponTemplate = (T = "BeowulfRifle_CV", P = "Requirements", Requirements = (RequiredTechs[0]="ModularWeapons", RequiredEngineeringScore=10, bVisibleIfPersonnelGatesNotMet=true, SpecialRequirementsFn=""))
++Edit_X2WeaponTemplate = (T = "BeowulfRifle_MG", P = "Requirements", Requirements = (RequiredTechs[0]="MagnetizedWeapons", RequiredEngineeringScore=10, bVisibleIfPersonnelGatesNotMet=true, SpecialRequirementsFn=""))
++Edit_X2WeaponTemplate = (T = "BeowulfRifle_BM", P = "Requirements", Requirements = (RequiredTechs[0]="PlasmaRifle", RequiredEngineeringScore=20, bVisibleIfPersonnelGatesNotMet=true, SpecialRequirementsFn=""))
```

Comment:
Bridge mods should not depend on other mods that aren't already dependencies of either Beowulf Rifles or Strategy Overhaul. Instead, accomplish these changes using an OnPostTemplatesCreated template modification instead.

Reply:
Removed Template Editor as dependency. Using OPTC instead :)